app/components/elements/button_component.rb

```diff
@@ -20,7 +20,7 @@ def call
         type: 'button',
         **options
       ) do
-        label
+        label || content
```

Comment:
Allow button component to have content injected. Allows us to style a button as a bootstrap icon, such as for deleting nested forms.